File: API/Helpers/UserParams.cs
namespace API.Helpers
{
    public class UserParams : PaginationParams
    {
        public string CurrentUsername { get; set; }
        public string SearchLocation { get; set; }
        public string SearchInstrument { get; set; }
        public string SortBy { get; set; } = "lastActive";
    }
}

File: client/src/app/_models/userParams.ts
import { User } from "./user";

export class UserParams {
    gender: string;
    pageNumber = 1;
    pageSize = 5;
    searchLocation = '';
    searchInstrument = '';
    searchExperience = '';
    sortBy = 'lastActive';

    constructor(user: User) {

    }
}